File: python/Length of Last Word.py
def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        l = len(s)
        if l==0:
            return 0;
        temp = s.split();
        if temp == []:
            return 0;
        res = temp[len(temp)-1]  
        
        
        return len(res)

File: java/Length of Last Word.java
public int lengthOfLastWord(String s) {
        
        if (s == null) {
            return 0;
        }
        int len = s.length()-1;
        int count =0;
        for(int i = len ; i >=0;i--) {
            if(s.charAt(i) != ' ') {
                count++;
            }
            if ( s.charAt(i) == ' ' && count != 0){
               return count;
            }
        }
        
        return count;
    }
